### PR TITLE
(v103) Bug 1779797 - Android: release branch string uplifts should auto-merge if CI is green

### PR DESCRIPTION
**Pull request by JohanLorenzo**
This just uplifts https://github.com/mozilla-mobile/fenix/pull/26073 and https://github.com/mozilla-mobile/fenix/pull/26046 to the v103 branch. No conflicts had to be resolved. More context in https://github.com/mozilla-mobile/fenix/pull/26073#issuecomment-1189146104

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [ ] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -16,11 +16,14 @@ tasks:
                 $if: 'event.sender.login == "bors[bot]"'
                 then: 'skaspari+mozlando@mozilla.com'   # It must match what's in bors.toml
```
Comment (ahal) [inline .taskcluster.yml:17]: Should this be removed while we're here?

File: taskcluster/ci/complete/kind.yml [hunk 2]
```diff
@@ -14,8 +14,15 @@ transforms:
     - taskgraph.transforms.code_review:transforms
```
Comment (ahal) [inline taskcluster/ci/complete/kind.yml:14]: Oh, maybe this transform already solves https://github.com/taskcluster/taskgraph/issues/71 :). I guess we could rename it and make it a bit more generic still.

**Review by ahal — APPROVED** [review]

File: .taskcluster.yml (hunk 1 above)
Comment (JohanLorenzo) [reply .taskcluster.yml:17]: No yet. I thought so but bors is still used to prune old branches. I plan to get rid of it soon 🙂

File: taskcluster/ci/complete/kind.yml (hunk 2 above)
Comment (JohanLorenzo) [reply taskcluster/ci/complete/kind.yml:14]: Oh yeah! 😃 I remember Tom Prince imported it in Taskgraph. The name was already set in gecko, IIRC.

**Comment by JohanLorenzo** [comment]
@mergifyio refresh

**Pull request merged**